test/hydrated_test.dart

```diff
@@ -75,6 +77,39 @@ void main() {
     /// clean up
     subject.close();
   });
+
+  test('Listener BehaviorSubject behavior', () async {
```

Comment:
What do you think about adding more context into the test description. For example: 
`HydratedSubject emits the latest value on listener added`.